velox/common/encode/Base64.cpp

```diff
@@ -22,27 +22,81 @@
 
 namespace facebook::velox::encoding {
 
+namespace {
+
+class IOBufWrapper {
```

Comment:
Can we keep IOBufWrapper at its old code location to minimize the diff?

Reply:
reverted the changes